### PR TITLE
Update esapi version in opensaml

### PR DESCRIPTION
**Pull request by Yasara-Y**
Update the esapi version embedded in opensaml to non-vulnerable v2.2.3.1 
Related to https://github.com/wso2/product-is/issues/12489

### CODE REVIEW

File: opensaml/3.3.1.wso2v3/pom.xml [hunk 1]
```diff
@@ -0,0 +1,1151 @@
+<!--
+  ~ Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (deshankoswatte) [inline opensaml/3.3.1.wso2v3/pom.xml:2]: Change year to 2021 and check if there is a new license file.

Comment (Yasara-Y) [reply opensaml/3.3.1.wso2v3/pom.xml]: Fixed.

**Review by vihanga-liyanage — APPROVED** [review]

File: opensaml/3.3.1.wso2v3/pom.xml [hunk 2]
```diff
@@ -0,0 +1,1151 @@
...
+                            @java-support-${shibboleth.version}.jar!/templates/**.vm,
+                            @java-support-${shibboleth.version}.jar!/schema/**.xsd,
+                            @java-support-${shibboleth.version}.jar!/conf/**,
+                        </Include-Resource>
+                    </instructions>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+    <properties>
```
Comment (AnuradhaSK) [inline opensaml/3.3.1.wso2v3/pom.xml:1103]: Better to leave a line between </build> and <properties>

Comment (Yasara-Y) [reply opensaml/3.3.1.wso2v3/pom.xml:1103]: Fixed with https://github.com/wso2/orbit/pull/492

**Pull request merged**